app/pandas.ipynb

```diff
@@ -182,16 +223,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 16,
-   "outputs": [],
+   "execution_count": 9,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "    Name  Age         City  Salary  BirthYear\n0   John   16     New York   50000       2007\n1  Emily   28  Los Angeles   60000       1995\n2   Ryan   22      Chicago   45000       2001",
+      "text/html": "<div>\n<style scoped>\n    .dataframe tbody tr th:only-of-type {\n        vertical-align: middle;\n    }\n\n    .dataframe tbody tr th {\n        vertical-align: top;\n    }\n\n    .dataframe thead th {\n        text-align: right;\n    }\n</style>\n<table border=\"1\" class=\"dataframe\">\n  <thead>\n    <tr style=\"text-align: right;\">\n      <th></th>\n      <th>Name</th>\n      <th>Age</th>\n      <th>City</th>\n      <th>Salary</th>\n      <th>BirthYear</th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>0</th>\n      <td>John</td>\n      <td>16</td>\n      <td>New York</td>\n      <td>50000</td>\n      <td>2007</td>\n    </tr>\n    <tr>\n      <th>1</th>\n      <td>Emily</td>\n      <td>28</td>\n      <td>Los Angeles</td>\n      <td>60000</td>\n      <td>1995</td>\n    </tr>\n    <tr>\n      <th>2</th>\n      <td>Ryan</td>\n      <td>22</td>\n      <td>Chicago</td>\n      <td>45000</td>\n      <td>2001</td>\n    </tr>\n  </tbody>\n</table>\n</div>"
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# Write your code here"
+    "df[\"BirthYear\"] = 2023 - df[\"Age\"]\n",
```

Comment:
Don't hardcode year value. You can use pd.to_datetime("now").year for getting current year